interface.go

```diff
@@ -38,6 +38,7 @@ type InterfaceConfig struct {
 	DropMulticast           bool
 	UDPBatchSize            int
 	routines                int
+	ConntrackCache          time.Duration
```

Comment:
The names feel like they collide here, `conntrackCache` being a type and an object while `ConntrackCache` is a duration. Move this one and the one below to `ConntrackCacheTimeout`?